### PR TITLE
Update sandbox

### PR DESCRIPTION
**Pull request by thisisdano**
[Preview](https://federalist-d5e7c07c-6ffa-4b8e-935f-49a7df24a505.app.cloud.gov/preview/uswds/uswds-sandbox/dw-update-sandbox/) →

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,4 @@
 _site
-_sass
```
Comment (thisisdano) [inline .gitignore:2]: Include _sass folder to allow proper builds on Federalist

File: _includes/banner.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
+<section class="usa-banner" aria-label="Official government website">
```
Comment (thisisdano) [inline _includes/banner.html:1]: Add the current banner since it's a common prototyping needs

File: _layouts/default.html [hunk 3]
```diff
@@ -5,7 +5,7 @@
 <html lang="en">
 {% include head.html %}
 
-<body>
+<body class="{{ page.body }}">
```
Comment (thisisdano) [inline _layouts/default.html:8]: Add a hook for adding body classes via page metadata

File: build.sh [hunk 4]
```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 
 if [ ! -d "node_modules/uswds/dist" ]; then
-  npm run uswds-install && npm run uswds-build
+  npm explore uswds -- npm run federalist
```
Comment (thisisdano) [inline build.sh:4]: Properly build `dist` directory if loading a non-release USWDS version

File: package.json [hunk 5]
```diff
@@ -4,7 +4,7 @@
   "author": "USWDS",
   "license": "CC0-1.0",
   "scripts": {
-    "federalist": "npm install && npm run uswds-copy-assets && npm run uswds-build-sass",
+    "federalist": "npm install && ./build.sh && npm run uswds-copy-assets && npm run uswds-build-sass",
```
Comment (thisisdano) [inline package.json:7]: Build dist directory on Federalist

**Review by mejiaj — APPROVED** [review]

**Comment by maya** [comment]
`assets` should be removed from `.gitignore` too since you already ignore `assets/uswds`. We need to add our own image assets in version control.

**Pull request merged**